src/elektra/windows/redshift-conf.c

```diff
@@ -0,0 +1,754 @@
+// clang-format off
```

Comment:
Is there a problem with the code generator under windows? Then it is probably better to either:

1. fix the problem with the code generator
2. always add the generated files (for any platform)

Reply:
Thanks for the hint! I have implemented 2. and modified the Makefile and CONTRIBUTING.md so that developers are aware they need to manually trigger the generation on changes of the specification files. I will open an issue in Elektra's repo for compatiblity of the code generator with Windows.